hw2/03-form.html

```diff
@@ -9,11 +9,36 @@
       rel="stylesheet"
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
     />
+    <link rel="stylesheet" href="03-form.css">
     <script defer src="03-form.js"></script>
   </head>
   <body class="bg-dark">
     <form class="bg-light border rounded w-50 mx-auto mt-5 p-3">
       <h1 class="mt-2 mb-4">Contact Form</h1>
+      <div class="form-group">
+        <label for="name">Name *</label>
+        <input type="text" class="form-control" id="name" required>
+      </div>
+      <div class="form-group">
+        <label for="email">Email *</label>
+        <input type="email" class="form-control" id="email" required>
+      </div>
+      <div class="form-group">
+        <label for="message">Submit your message:</label>
+        <textarea type="text" class="form-control" id="message"></textarea>
+      </div>
+      <div class="form-check">
+        <input type="checkbox" class="form-check-input" id="newsletter"></input>
+        <label for="newsletter">Sign up for the newsletter</label>
+      </div>
+      <div class="btn-toolbar mt-3" role="toolbar" aria-label="Toolbar with button groups">
+        <div class="btn-group" role="group" aria-label="Submit button group">
+          <button type="button" class="btn btn-primary" id="submit">Submit</button>
```

Comment:
its good practise to have `type = "submit"` button for the `form` element, so that basic validation will be done by browsers.